### ISSUE TITLE
Accessibility: Color Contrast

### ISSUE THREAD

**Issue by Mackenzie-rae** [PavlAvstin/OZ-CSC-480-HCI-521-Fall-2023-#190]
When using the wave plugin tool it was found under the rating movies modal there were 5 instances where the color contrast failed.    The back button, and the four public buttons failed. The contrast ratio for these buttons are 3.59:1. When passing for this tool has usually been 8.59:1.
